contracts/UFragments.sol

```diff
@@ -358,12 +358,7 @@ contract UFragments is ERC20Detailed, Ownable {
      */
     function decreaseAllowance(address spender, uint256 subtractedValue) external returns (bool) {
         uint256 oldValue = _allowedFragments[msg.sender][spender];
-        uint256 newValue;
-        if (subtractedValue >= oldValue) {
-            newValue = 0;
-        } else {
-            newValue = oldValue.sub(subtractedValue);
-        }
+        uint256 newValue = subtractedValue >= oldValue ? 0 : oldValue.sub(subtractedValue);
```

Comment:
Minor, but I like the style that puts parens around the ternary operator's condition clause.